views/contributor/summary.templ

```diff
@@ -10,11 +10,13 @@ import (
 )
 
 type SummaryArgs struct {
-	Role             string
-	Contributors     []*models.Contributor
-	URL              string
-	URLTarget        string
-	CurrentUserRoles string
+	Role                    string
+	Contributors            []*models.Contributor
+	URL                     string
+	URLTarget               string
+	CurrentUserRoles        string
+	CanViewMoreContributors bool
+	CanEditContributors     bool
```

Comment:
Not the biggest fan of these 2 new flags, but don't know how we can do this in another manner when the contributor summary is not even aware of it's surrounding context (either publication or dataset).